Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2781,30 +2897,61 @@ private protected virtual void OnDisable()
     PinnedList<Color>   m_tmpC  = new PinnedList<Color>();
-    
+
 #if AT_USE_SPLINES
     PinnedList<float3> m_tmpFloat3 = new PinnedList<float3>();
 #endif
 
-//----------------------------------------------------------------------------------------------------------------------
-    
-    // keyword strings
-    const string _Color   = "_Color";
-    const string _MainTex = "_MainTex";
+        //----------------------------------------------------------------------------------------------------------------------
+
+        // keyword strings
+        internal const string _Color = "_Color";
+        internal const string _BaseColor = "_BaseColor";
+        internal const string _MainTex = "_MainTex";
+        internal const string _BaseColorMap = "_BaseColorMap";
+
+        internal const string _BaseMap = "_BaseMap";
+        internal const string _GlossMap = "_GlossMap";
+
+        private const string _EmissionColor         = "_EmissionColor";
+        private const string _EmissionMap           = "_EmissionMap";
+        private const string _EmissiveColorMap      = "_EmissiveColorMap";
+        private const string _EMISSION              = "_EMISSION";
+        private const string _EmissiveIntensity     = "_EmissiveIntensity";
+        private const string _EmissiveIntensityUnit = "_EmissiveIntensityUnit";
+        private const string _EmissiveColorLDR      = "_EmissiveColorLDR";
+        private const string _EmissiveColor         = "_EmissiveColor";
+        private const string _EmissionStrength      = "_EmissionStrength";
+        private const string _UseEmissiveIntensity  = "_UseEmissiveIntensity";
+
+        internal const string _Metallic = "_Metallic";
+        internal const string _Glossiness = "_Glossiness";
+        internal const string _Smoothness = "_Smoothness";
+
+        internal const string _MetallicGlossMap = "_MetallicGlossMap";
+        internal const string _METALLICGLOSSMAP = "_METALLICGLOSSMAP";
+        internal const string _METALLICSPECGLOSSMAP = "_METALLICSPECGLOSSMAP";
+
+        private const string _BumpMap = "_BumpMap";
+        private const string _NORMALMAP = "_NORMALMAP";
+        private const string _NormalMap = "_NormalMap";
+
+        private const string _ParallaxMap = "_ParallaxMap";
+        private const string _PARALLAXMAP = "_PARALLAXMAP";
+        private const string _Parallax    = "_Parallax";
+        private const string _HeightMap   = "_HeightMap";
+        private const string _HEIGHTMAP   = "_HEIGHTMAP";
+
+        private const string _ALPHATEST_ON = "_ALPHATEST_ON";
+        private const string _AlphaClip = "_AlphaClip";
+        private const string _AlphaCutoffEnable = "_AlphaCutoffEnable";
 
-    const string _EmissionColor = "_EmissionColor";
-    const string _EmissionMap   = "_EmissionMap";
-    const string _EMISSION      = "_EMISSION";
+        internal const string _MaskMap = "_MaskMap";
+        internal const string _MASKMAP = "_MASKMAP";
```

Comment:
Same with moving these out to a constants class. 

Also, what is the difference between _MASKMAP and _MaskMap?

Reply:
I've moved them to MeshSyncConstants.
The capital ones are the shader keywords.